deploy/olm-catalog/ibm-block-csi-operator/1.10.0/manifests/ibm-block-csi-operator.clusterserviceversion.yaml

```diff
@@ -221,6 +221,44 @@ spec:
           - description: The current version of the driver.
             displayName: Version
             path: version
+      - name: hostdefiners.csi.ibm.com
```

Comment:
what about adding an example to the `alm-examples`?

Reply:
done